massa-ledger/src/ledger_db.rs

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+use massa_hash::Hash;
+use massa_models::Address;
+use rocksdb::{ColumnFamilyDescriptor, Options, WriteBatch, DB};
+
+use crate::{ledger_changes::LedgerEntryUpdate, LedgerEntry, SetOrDelete, SetOrKeep};
+
+const DB_PATH: &str = "../.db";
```

Comment:
The other DB for consensus is save in the storage folder of `massa-node`. I think we should group them.